tests/pbkdf2/Makefile

```diff
@@ -8,6 +8,9 @@ USEMODULE += base64
 # Use a terminal that does not introduce extra characters into the stream.
 RIOT_TERMINAL ?= socat
 
+#ensure the rx buffer has some room even with large test patterns
+CFLAGS += -DSTDIO_UART_RX_BUFSIZE=128
```

Comment:
(some boards have issues with larger uart transfers, like, the samr21-xpro over it's edbg)

Reply:
But that isn't an issue of the stdio-buffer-size being to big --i think

Reply:
In the pbkdf2-test are transfers of upto 67 byte (65+2) only limited by the speed of the serial connection. 
The default rx buffer is 64 bytes -> if the uart isr keep coming in and hence the bytes not processed, it is getting tight in that buffer.